export class CalledExistsError extends Error {
  constructor() {
    super('Called exists')
  }
}
